tests/unit/test_unit_metobs.py

```diff
@@ -422,7 +422,7 @@ def test_unit_metobsparameterv1_init(
 
 
 class TestUnitMetObsStationV1:
```

Comment:
Shouldn't these names follow suit?

Reply:
Fixed in [8fabb13](https://github.com/Ingenjorsarbete-For-Klimatet/ifk-smhi/pull/42/commits/8fabb135cb4a7a04d8922c2d9dd0b18d60ad0866).